convert/src/test/kotlin/com/github/zeldigas/text2confl/convert/PageContentTest.kt

```diff
@@ -70,6 +71,7 @@ class PageContentTest {
 
     @Test
     internal fun `Invalid result for unbalanced xml`() {
+        Locale.setDefault(Locale.ENGLISH);
```

Comment:
this is a bad idea - especially taking into account that you do it without any try/finally/extension way.

It's better to use something like this https://wesome.org/junit-5-pioneer-defaultlocale or create adhoc extension in tests to do something similar